damnit/gui/main_window.py

```diff
@@ -533,6 +533,10 @@ def handle_update(self, message):
             )
         elif msg_kind == MsgKind.variable_set:
             self.table.handle_variable_set(data)
+            # update tag filtering
+            self.table_view.apply_tag_filter(
+                self.table_view._current_tag_filter
+            )
```

Comment:
I think that it would be nice to inform the user about the update, given that is automatic. A simple message in the status box or also through some change of the filter button (little dot or changing the color).

Reply:
I think I'll go against that one. Tagging a variable will happen 99.9% of the time while developing the context file, particularly not while using the GUI. And I can't think of a particularly useful hint for that change. For the visible filters, we already can see the list of variable change and for hidden tags, that's probably not a useful information.
We can of course revisit that once the feature is used if necessary.

Reply:
That's fair. Probably the only case in which a more explicit message would be useful would be when something is actually selected. I guess that while developing one doesn't really filter the variables. My only "concern" is that one gets confused as the shown variables change without an apparent reason (for example if one uses the interface while someone else changes stuff). But I guess that we can wait and see if this is actually confusing for users.